### PR TITLE
add: Fee quote

### PR DESCRIPTION
**Pull request by GulfOfAlaska**

### CODE REVIEW

**Review by stevenkhong — APPROVED** [review]

File: src/clients/HydrogenClient.ts [hunk 1]
```diff
@@ -4,13 +4,17 @@ import { BlockchainUtils } from "@carbon-sdk/util";
 import { EnumMember } from "typescript";
```
Comment (sarah-thong) [inline src/clients/HydrogenClient.ts:4]: This type is not used, so do remove it.

File: src/clients/HydrogenClient.ts [hunk 2]
```diff
@@ -106,6 +120,21 @@ class HydrogenClient {
       data: result.data.map(formatCrossChainTransferDetailed),
     }
   }
+
+  async getFeeQuote(req: GetFeeQuoteRequest): Promise<GetFeeQuoteResponse> {
+    this.checkState();
```
Comment (sarah-thong) [inline src/clients/HydrogenClient.ts:125]: This line needs to be commented out/removed as this throws an error if user is connected to Testnet/Localhost.

File: src/clients/HydrogenClient.ts [hunk 3]
```diff
@@ -106,6 +120,21 @@ class HydrogenClient {
...
+    const request = this.apiManager.path('fee_quote', {}, {
+      ...req,
+      include_tx: true,
```
Comment (sarah-thong) [inline src/clients/HydrogenClient.ts:128]: `include_tx` is not a param in the fee_quote endpoint

File: src/clients/HydrogenClient.ts [hunk 4]
```diff
@@ -106,6 +120,21 @@ class HydrogenClient {
...
+    })
+    const response = await request.get();
+    const result = response.data;
+
+    return {
+      ...result,
+      data: result.data.map(formatFeeQuote),
```
Comment (sarah-thong) [inline src/clients/HydrogenClient.ts:135]: The json data returned from the fee_quote endpoint is not an array.
Screenshot 2022-08-29 at 3 13 37 PM

So line 133 - 136 should be changed to this instead:
```
return formatFeeQuote(result)
```

File: src/clients/HydrogenClient.ts [hunk 5]
```diff
@@ -4,13 +4,17 @@ import { BlockchainUtils } from "@carbon-sdk/util";
 import { EnumMember } from "typescript";
 import { ChainTransaction, CrossChainTransfer, CrossChainTransferDetailed, GetDetailedTransfersResponse, GetStatsResponse, GetTransfersRequest, GetTransfersResponse } from "../hydrogen";
 import dayjs from "dayjs";
+import { FeeQuote, GetFeeQuoteRequest, GetFeeQuoteResponse } from "@carbon-sdk/hydrogen/feeQuote";
```
Comment (sarah-thong) [inline src/clients/HydrogenClient.ts:8]: Do add the Hydrogen Testnet URL to the networkConfig file (`src/constant/network.ts`). It should added to `hydrogenUrl` under the network config of `CarbonSDK.Network.Testnet`.
Screenshot 2022-08-29 at 3 24 12 PM

**Pull request merged**